src/lib/nodes/utils/getHealthcheckConfig.ts

```diff
@@ -49,3 +50,16 @@ export function getHealthCheckInterval(nodeLabel: TNodeLabel, interval: Healthch
       )
   }
 }
+
+/**
+ * Get base URL for requests depending on availability of a node's domain.
+ * @param { Node } node A node instance.
+ * @returns { string } Base URL.
+ */
+export function getBaseURL(node: Node): string {
+  const baseURL = node.preferDomain ? node.url : (node.altIp as string)
+
+  console.info({ baseURL, altIp: node.altIp, url: node.url })
+
+  return baseURL
+}
```

Comment:
I don't see a need for this helper. All that logic belongs to the Node instance class and can be kept there.

Reply:
https://github.com/Adamant-im/adamant-im/pull/768/commits/69e755669fdbc5ceca1a779df479aa5dcfa91627